tests/test_datamodel.py

```diff
@@ -880,5 +883,43 @@ def test_precision(self) -> None:
             Column(**{"name": "testColumn", "@id": "#test_col_id", "datatype": "double", "precision": 6})
 
 
+class SchemaSerializationTest(unittest.TestCase):
+    """Test serialization and deserialization of the schema data model."""
+
+    def test_serialization(self) -> None:
+        """Test serialization of the schema data model."""
+        # Read the original YAML content from the test_serialization.yaml file
+        with open(TEST_SERIALIZATION) as file:
+            original_yaml_content = file.read()
+
+        # Load the schema from the original YAML content
+        schema_out = Schema.from_uri(TEST_SERIALIZATION)
+        serialized_data = schema_out.model_dump(by_alias=True, exclude_none=True, exclude_defaults=True)
+
+        # Write the serialized data to a temporary YAML file
+        with tempfile.NamedTemporaryFile(delete=False, suffix=".yaml", mode="w+") as temp_file:
+            print(f"Dumping schema to {temp_file.name}")
```

Comment:
Did you intend to leave the debugging prints in this test?  I don't usually find them that useful, since pytest gathers up the stdout and shows it out of order from other test output.

Reply:
I want to print the differences between the original and reread files if there are any, but this particular print out isn't useful and can be removed.